addons/mail/static/src/xml/web_kanban_activity.xml

```diff
@@ -2,8 +2,8 @@
 <templates xml:space="preserve">
 
 <t t-name="mail.KanbanActivity">
-    <div class="o_kanban_inline_block dropdown o_mail_activity">
-        <a class="dropdown-toggle o-no-caret o_activity_btn" data-boundary="viewport" data-flip="false" data-toggle="dropdown" role="button">
+    <div class="o_kanban_inline_block dropdown o_mail_activity position-relative">
+        <a class="dropdown-toggle o-no-caret o_activity_btn" data-boundary="none" data-flip="false" data-toggle="dropdown" role="button">
```

Comment:
Either way the horizontal positioning is broken; I tried to open the dropdown menu on mobile and it popped off my screen. Because of this, there is no choice but to keep the boundary, I guess 😕

Reply:
> Do you know about `data-display="static"`? From [what I read](https://stackoverflow.com/a/49323938), it seems to synthesise both behaviours (`position-relative` + `data-boundary="none"`).

I already tried but it didn't work, now it does, I must have mixed with some other tags.


> Either way the horizontal positioning is broken; I tried to open the dropdown menu on mobile and it popped off my screen. Because of this, there is no choice but to keep the boundary, I guess 😕

Concerning the mobile view, I tried and did not spot any issues concerning the dropdown (when using `data-display="static"`)

Reply:
Finally fixed, checked on my MAC, Linux and Mobile.